pom.xml

```diff
@@ -150,12 +130,12 @@
                                     <goal>package-for-production</goal>
                                 </goals>
                                 <configuration>
-                                    <transpileOutputDirectory>${webapp.directory}</transpileOutputDirectory>
+                                    <transpileOutputDirectory>${transpilation.directory}</transpileOutputDirectory>
```

Comment:
Ideally maven plugin could detect that this is a spring boot project and have these as defaults.